pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -263,8 +263,9 @@ const (
 )
 
 type LoadBalancerAccessSpec struct {
```

Comment:
One last nitpick ... should have mentioned it earlier.  Can we get some code level docs?  Once we generate the api server docs they will show up like: https://github.com/kubernetes/kops/blob/master/docs/apireference/build/index.html ~ we need to get that hosted.

Here is a good guide, https://blog.golang.org/godoc-documenting-go-code, but I assume that you are an experienced gopher.

Docs here and in cluster would be AWESOME!!